sample-apps/endoscopy/lib/configs/inbody.py

```diff
@@ -33,7 +33,7 @@ class InBody(TaskConfig):
     def init(self, name: str, model_dir: str, conf: Dict[str, str], planner: Any, **kwargs):
         super().init(name, model_dir, conf, planner, **kwargs)
 
-        bundle_name = "endoscopic_inbody_classification"
+        bundle_name = conf.get("bundle_name", "endoscopic_inbody_classification")
         bundle_version = "0.2.0"
```

Comment:
in that case, you also take bundle_version (similar to bundle_name).. name alone is not enough in this case..

Reply:
Sure, thanks!